src/packages/auth-ui-components/src/components/password/reset-password/component.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import { Form, Formik, FormikHelpers } from 'formik';
+import { useMutation } from '@apollo/client';
+import { GraphweaverLogo, Alert, Button } from '@exogee/graphweaver-admin-ui-components';
+
+import styles from './styles.module.css';
+
+import { RESET_PASSWORD } from './graphql';
+import { ConfirmComponent, PasswordComponent } from '../password/component';
+
+interface Form {
+	password: string;
+	confirm: string;
+}
+
+export const ResetPassword = () => {
+	const [resetPassword] = useMutation<{ result: boolean }>(RESET_PASSWORD);
+	const [error, setError] = useState<Error | undefined>();
+	const [searchParams] = useSearchParams();
+	const navigate = useNavigate();
+
+	const token = searchParams.get('token');
+	const redirectUri = searchParams.get('redirect_uri');
+
+	const handleOnSubmit = async (values: Form, { resetForm }: FormikHelpers<Form>) => {
+		setError(undefined);
+
+		if (values.password !== values.confirm) {
+			setError(new Error('Passwords do not match'));
+			return;
+		}
+
+		try {
+			const { data } = await resetPassword({
+				variables: {
+					password: values.password,
+					token,
+				},
+			});
+
+			const redirectUrl = redirectUri ?? new URL('/');
+			navigate(`/auth/login?redirect_uri=${redirectUrl}`, { replace: true });
```

Comment:
Is this path guaranteed to be here?

Reply:
Guaranteed how?

Reply:
As in is it possible that the developer sets their login page up at `/log-me-in` instead of `/auth/login`?

Reply:
ah I see what you mean!

No is the answer 🤔

Reply:
Moved the values to env vars 👍